test/fixtures/test-csv-data.js

```diff
@@ -451,94 +451,98 @@ export const wktCsv = `a_zip,simplified_shape_v2,simplified_shape,m_rate,c_zip_t
 7023,"{""type"":""LineString"",""coordinates"":[[-74.387589,40.632238],[-74.387589,40.632238]]}","LINESTRING (-74.387589 40.632238, -74.387589 40.632238)",7.6,C_Medium_High,29.2
 `;
 
-// csv data
+// output of processCsvData
 export const testFields = [
   {
     type: 'timestamp',
+    fieldIdx: 0,
     name: 'gps_data.utc_timestamp',
     format: 'YYYY-M-D H:m:s',
-    tableFieldIndex: 1,
-    analyzerType: 'DATETIME'
+    analyzerType: 'DATETIME',
+    valueAccessor: values => values[0]
   },
   {
     type: 'real',
+    fieldIdx: 1,
     name: 'gps_data.lat',
     format: '',
-    tableFieldIndex: 2,
-    analyzerType: 'FLOAT'
+    analyzerType: 'FLOAT',
+    valueAccessor: values => values[1]
   },
   {
     type: 'real',
+    fieldIdx: 2,
     name: 'gps_data.lng',
     format: '',
-    tableFieldIndex: 3,
-    analyzerType: 'FLOAT'
+    analyzerType: 'FLOAT',
+    valueAccessor: values => values[2]
   },
   {
     type: 'string',
+    fieldIdx: 3,
     name: 'gps_data.types',
     format: '',
-    tableFieldIndex: 4,
-    analyzerType: 'STRING'
+    analyzerType: 'STRING',
+    valueAccessor: values => values[3]
   },
   {
     type: 'timestamp',
+    fieldIdx: 4,
     name: 'epoch',
     format: 'X',
-    tableFieldIndex: 5,
-    analyzerType: 'TIME'
+    analyzerType: 'TIME',
+    valueAccessor: values => values[4]
   },
   {
     type: 'boolean',
+    fieldIdx: 5,
     name: 'has_result',
     format: '',
-    tableFieldIndex: 6,
-    analyzerType: 'BOOLEAN'
+    analyzerType: 'BOOLEAN',
+    valueAccessor: values => values[5]
   },
   {
     type: 'integer',
+    fieldIdx: 6,
     name: 'id',
     format: '',
-    tableFieldIndex: 7,
-    analyzerType: 'INT'
+    analyzerType: 'INT',
+    valueAccessor: values => values[6]
   },
   {
     type: 'timestamp',
+    fieldIdx: 7,
     name: 'time',
     format: 'YYYY-M-DTHH:mm:ss.SSSS',
-    tableFieldIndex: 8,
-    analyzerType: 'DATETIME'
+    analyzerType: 'DATETIME',
+    valueAccessor: values => values[7]
   },
   {
     type: 'timestamp',
+    fieldIdx: 8,
     name: 'begintrip_ts_utc',
     format: 'YYYY-M-D HH:mm:ssZZ',
-    tableFieldIndex: 9,
-    analyzerType: 'DATETIME'
+    analyzerType: 'DATETIME',
+    valueAccessor: values => values[8]
   },
   {
     type: 'timestamp',
+    fieldIdx: 9,
     name: 'begintrip_ts_local',
     format: 'YYYY-M-D HH:mm:ssZZ',
-    tableFieldIndex: 10,
-    analyzerType: 'DATETIME'
+    analyzerType: 'DATETIME',
+    valueAccessor: values => values[9]
```

Comment:
this doesn't seem to be right,this should return a ts valueAccessor should return epoch milisecond, are these really tested?